build/MSBuildExtensions.targets

```diff
@@ -115,6 +118,23 @@ Copyright (c) .NET Foundation. All rights reserved.
     <LatestPatchVersionForNetCore2_0 Condition="'%24(LatestPatchVersionForNetCore2_0)' == ''">2.0.9</LatestPatchVersionForNetCore2_0>
     <LatestPatchVersionForNetCore2_1 Condition="'%24(LatestPatchVersionForNetCore2_1)' == ''">2.1.2</LatestPatchVersionForNetCore2_1>
   </PropertyGroup>
+  <ItemGroup>
+    <KnownFrameworkReference Include="Microsoft.Windows.UI"
+                              RuntimeFrameworkName="Microsoft.Private.WPF"
+                              DefaultRuntimeFrameworkVersion="$(WinFormsAndWpfVersion)"
```

Comment:
When do these three versions don't match?

Reply:
Once we have separate targeting packs and runtime packs, the targeting pack version could be different from the runtime framework version (the targeting pack will generally not be patched).  The default vs the latest runtime framework version will be the different versions depending on whether you roll forward to the latest runtime patch or not (generally because you are self-contained).